src/factom-event-emitter.js

```diff
@@ -321,6 +437,28 @@ class FactomEventEmitter extends EventEmitter {
             }
         });
     }
+
+    // Emit new pending factoid transactions for user-defined FCT address
+    _emitPendingFactoidTransaction(fctAddress, transactions) {
+        this.emit(`newPendingTransaction:${fctAddress}`, transactions);
+    }
+
+    /**
+     * Fetch pending FCT transactions for a given FCT address
+     * @param {string} - address
+     * @returns {array} - Array of pending FCT transactions
+     */
+    async getPendingTransactions(address) {
+        try {
+            const pendingTransactionsResponse = await this._cli.factomdApi('pending-transactions', {
```

Comment:
direct return without the intermediate variable will save us 2 lines! (and you can then even remove async)